README.md

```diff
@@ -224,6 +224,51 @@ See [using Ansible collections](https://docs.ansible.com/ansible/devel/user_guid
    > [!NOTE]
    Add this certificate to all RHTAS client nodes that use the `cosign` and `gitsign` binaries for signing and verifying artifacts.
 
+## Configuring a User-Provisioned MariaDB and Redis Instance for the RHTAS Ansible Collection
```

Comment:
We want to use sentence case for titles, the exception being proper nouns.

`Configuring a User-Provisioned MariaDB and Redis Instance for the RHTAS Ansible Collection` -->
`Configuring a user-provisioned MariaDB and Redis instance for the RHTAS Ansible Collection`